solidity/app/cl_utils.js

```diff
@@ -0,0 +1,35 @@
+const clWallet = require('./cl_wallet.js')
+
+const Eth = require('ethjs')
+
+global.clUtils = global.clUtils || {
+  personalAccount: '0x9CA9d2D5E04012C9Ed24C0e513C9bfAa4A2dD77f',
+  toWei: function toWei (eth) {
+    return (parseInt(eth.toString(), 10) * 10 ** 18).toString()
+  },
+  getTxReceipt: function getTxReceipt (txHash) {
```

Comment:
I think it creates more helpful output in stack traces. Instead of anonymous functions, the function name on the right is displayed.

Reply:
Looked it up and it seems that JS has improved since and we don't need the double name anymore:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/name

The key will automatically be assigned to the function.

Reply:
Good to know. Let's clean that up in a separate PR?